prisma/schema.prisma

```diff
@@ -190,23 +190,23 @@ enum Aid {
   SCOOTER
   WALKER
 
-  @@map("aid")
+   @@map("aid")
```

Comment:
Not sure why these spaces are being appended. Can you run `prisma format` before committing? That should prevent these spaces from being added.